packages/renderer/src/lib/pod/PodCreateFromContainers.svelte

```diff
@@ -14,6 +14,7 @@ import Fa from 'svelte-fa';
 import SolidPodIcon from '../images/SolidPodIcon.svelte';
 import Button from '../ui/Button.svelte';
 import type { PodCreatePortOptions } from '../../../../main/src/plugin/dockerode/libpod-dockerode';
+import Input from '../ui/Input.svelte';
```

Comment:
probably need to use `/@/` format for the import rather than `../..`

Reply:
Updated.